_posts/2023-04-04-s3-sns-notification.markdown

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: Pain point of S3 Notification Configuration for SNS
+description: !
+date: 2023-03-21
+tags: [S3 Notification Configurations, SNS Topic, Lambda Event]
+author: Nadia Reyhani
+image: /img/sns-Notifications-reconfiguration/SNS_Notification_Error.png
+---
+
+![Avoid "Unable to validate the following destination configurations" Error](/img/sns-Notifications-reconfiguration/SNS_Notification_Error.png)
+
+## How to Avoid "Unable to validate the following destination configurations" Error
+
+As is widely known, working with CloudFormation as an Infrastructure as Code tool to deploy resources poses a major challenge when it comes to deciphering errors in the create/update stack process. These errors are not usually straightforward or informative, which makes it difficult to identify the problem immediately. Recently, I had a good fun with reconfigure notifications on an Amazon S3 bucket, and it took me a full day to trouble shoot why an error was occurring and how to resolve it. I consulted numerous resources, each of which pointed to a specific reason for the deployment failure. Ultimately, I had to combine all the information I had gathered to successfully address the issue.
```

Comment:
`Recently, I had a good fun with reconfigure notifications`

TO:
`Recently, I had good fun with re-configuring notifications`